web/client/translations/data.en-US.json

```diff
@@ -577,6 +577,9 @@
             "redoBtnTooltip": "Go forward"
         },
         "infoFormatLbl": "Identify response format",
+        "infoTriggerLabel": "Identify trigger",
```

Comment:
All the translation sounds like "Identify the trigger", try to use "Trigger event for Identify " as initial translation file.
italian: "Evento trigger interrogazioni su mappa"

Reply:
Fixed